enterprise/app/repo/repo.tsx

```diff
@@ -149,20 +155,44 @@ export default class RepoComponent extends React.Component<RepoComponentProps, R
     );
   }
 
+  getSecrets() {
+    let s = this.props.search.get("secret") || this.props.search.get("secrets") || "";
+    return s.split(",");
```

Comment:
`"".split(",")` returns `[""]` but want `[]`

```suggestion
    return s ? s.split(",") : [];
```